File: gluecodium/src/test/resources/smoke/skip/output/android/com/example/smoke/SkipFunctions.java
/*

 *
 */

package com.example.smoke;

import android.support.annotation.NonNull;
import com.example.NativeBase;

public final class SkipFunctions extends NativeBase {

    /**
     * For internal use only.
     * @hidden
     * @param nativeHandle The SDK nativeHandle instance.
     * @param dummy The SDK dummy instance.
     */
    protected SkipFunctions(final long nativeHandle, final Object dummy) {
        super(nativeHandle, new Disposer() {
            @Override
            public void disposeNative(long handle) {
                disposeNativeHandle(handle);
            }
        });
    }

    private static native void disposeNativeHandle(long nativeHandle);



    public static native boolean notInSwift(final boolean input);


    public static native float notInDart(final float input);


    @NonNull
    public static native String notInKotlin(@NonNull final String input);



}

File: gluecodium/src/test/resources/smoke/skip/output/android-kotlin/com/example/smoke/SkipFunctions.kt
/*

 *
 */

@file:JvmName("SkipFunctions")

package com.example.smoke

import com.example.NativeBase

class SkipFunctions : NativeBase {



    /**
     * For internal use only.
     * @suppress
     * @param nativeHandle The handle to resources on C++ side.
     * @param tag Tag used by callers to avoid overload resolution problems.
     */
    protected constructor(nativeHandle: Long, @Suppress("UNUSED_PARAMETER") tag: Any?)
        : super(nativeHandle, { disposeNativeHandle(it) }) {}







    companion object {
        @JvmStatic private external fun disposeNativeHandle(nativeHandle: Long)


        @JvmStatic external fun notInJava(input: String) : String


        @JvmStatic external fun notInSwift(input: Boolean) : Boolean


        @JvmStatic external fun notInDart(input: Float) : Float
    }
}

